app/src/renderer/components/wallet/PageWallet.vue

```diff
@@ -125,6 +83,9 @@ export default {
       }
       return balances
     },
+    allBalances() {
```

Comment:
why don't we use just `this.wallet.balances` instead of this func ?

Reply:
Mainly for testing, so I don't have to mock wallet if I need just this.

Reply:
you currently still need to provide `wallet.balances` as `allDenomBalances` requires it. either use `this.allBalances` in there or use `wallet.balances` imo.